tests/parametric/test_parametric_workflow.py

```diff
@@ -22,7 +22,10 @@ def test_parametric_workflow():
     Path(pyfluent.EXAMPLES_PATH).mkdir(parents=True, exist_ok=True)
     tmp_save_path = tempfile.mkdtemp(dir=pyfluent.EXAMPLES_PATH)
     import_file_name = examples.download_file(
-        "Static_Mixer_main.cas.h5", "pyfluent/static_mixer", save_path=tmp_save_path
+        "Static_Mixer_main.cas.h5",
+        "pyfluent/static_mixer",
+        save_path=tmp_save_path,
+        return_without_path=False,
```

Comment:
Looking at this API now, it is probably simpler if the function returns just a Path object. `Path.stem` or `Path.absolute()` can be used later as per requirements.

Can be considered in a separate PR.

Reply:
Yes, I agree.